wiring/inc/spark_wiring_print.h

```diff
@@ -127,12 +127,20 @@ size_t Print::print(T n, int base)
         size_t t = 0;
         using PrintNumberType = typename PrintNumberTypeSelector<T>::type;
         PrintNumberType val;
+// FIXME: avoids 'comparison of constant '0' with boolean expression is always false'
+#if __GNUC__ >= 9
```

Comment:
Adding a separate `bool` specialization brings its own set of problems, so for now we are disabling this warning here specifically as a workaround.